content/5_Plat/Centroid.mdx

```diff
@@ -84,6 +84,8 @@ int main() {
 
 	get_subtree_size(0);
 	cout << get_centroid(0) + 1 << endl;
+
+	return 0;
```

Comment:
wait how does this make it easier to read

Reply:
This is a solution to the problem https://cses.fi/problemset/task/2079/ . Some judges require return 0 from the main function, so I decided to add this line to make the code more safe.

Reply:
which judge?

https://cplusplus.com/forum/beginner/14644/

> C++ specifically allows you to fall off the main() function without explicitly returning anything. The compiler is required to automagically insert a return 0; in there.

Reply:
I belive UVA used to require return 0, but I just checked it out and turns out this may really be optional, so I will remove this line.